pkg/verify/verify.go

```diff
@@ -200,8 +222,13 @@ func (v *ReleaseVerifier) Signatures() map[string][][]byte {
 	return copied
 }
 
+// AddStore adds additional stores for signature verification.
+func (v *releaseVerifier) AddStore(additionalStore store.Store) {
+	v.store = &serial.Store{Stores: []store.Store{additionalStore, v.store}}
```

Comment:
do we always want to add with a serial prepend?  I think we might want a getter/setter method pair so folks can pick the combination on the caller-side.

Reply:
This was just lifted exactly from CVO's current setting of 'verifier.Store` but also fine to go with the more flexible approach as well.

Reply:
That's the only usage I've noticed. I'd suggest you guys have a closer look at that package implementation and reconsider the overall shape, imo. For now I'd go with as is, and I'd definitely discourage getters & setters.

Reply:
As discussed with @wking, this will satisfy CVO's needs now and we can change it later if necessary.